python/tank/descriptor/io_descriptor/git.py

```diff
@@ -43,17 +56,109 @@ def __init__(self, descriptor_dict):
         # Note: the git command always uses forward slashes
         self._sanitized_repo_path = self._path.replace(os.path.sep, "/")
 
-    def _clone_repo(self, target_path):
+    @LogManager.log_timing
+    def _clone_then_execute_git_command(self, target, commands):
```

Comment:
Ah yes - it started off as a single command and i didn't update the method name, only the `commands` argument name...